StripeIdentity/StripeIdentityTests/Snapshot/PhoneOtpViewControllerSnapshotTest.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  PhoneOtpViewControllerSnapshotTest.swift
+//  StripeIdentityTests
+//
+//  Created by Chen Cen on 6/16/23.
+//
+
+import Foundation
+import iOSSnapshotTestCase
+
+@testable import StripeIdentity
+
+final class PhoneOtpViewControllerSnapshotTest: FBSnapshotTestCase {
+
+    override func setUp() {
+        super.setUp()
+
+        //        recordMode = true
```

Comment:
leftover

Reply:
this is a common pattern for our snapshot test, please see [this](https://github.com/search?q=repo%3Astripe%2Fstripe-ios%20recordMode%20%3D%20true&type=code) for example